src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/UpgradeAssistantHostExtensions.cs

```diff
@@ -172,6 +180,8 @@ private static void AddUserInput(this IServiceCollection services)
 
             const string LogFilePath = "upgrade-assistant.clef";
 
+            Console.WriteLine(options);
```

Comment:
what are you trying to log here? If you want to output something, use `_logger.Log()` calls

Reply:
Removed, not needed